libafl_nyx/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "libafl_nyx"
+version = "0.1.0"
+edition = "2021"
+authors = ["syheliel <syheliel@gmail.com>"]
+description = "libafl using nyx, only avaliable on linux"
+documentation = "https://docs.rs/libafl_nyx"
+repository = "https://github.com/AFLplusplus/LibAFL/"
+readme = "../README.md"
+license = "MIT OR Apache-2.0"
+keywords = ["fuzzing", "testing", "security"]
+categories = ["development-tools::testing", "emulators", "embedded", "os", "no-std"]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[target.'cfg(target_os = "linux")'.dependencies]
+libnyx = {git = "https://github.com/nyx-fuzz/libnyx.git"}
```

Comment:
Maybe we want to add the commit hash (`rev =`) to avoid breakages in the future

Reply:
fixed